static/app/views/replays/filters.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import DatePageFilter from 'sentry/components/datePageFilter';
+import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import ProjectPageFilter from 'sentry/components/projectPageFilter';
+import space from 'sentry/styles/space';
+
+function ReplaysFilters() {
+  return (
+    <FilterContainer>
+      <PageFilterBar>
+        <ProjectPageFilter />
+        <EnvironmentPageFilter alignDropdown="left" />
+        <DatePageFilter alignDropdown="left" />
+      </PageFilterBar>
+    </FilterContainer>
+  );
+}
+
+const FilterContainer = styled('div')`
+  width: max-content;
+  max-width: 100%;
+  margin-bottom: ${space(1)};
+
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    grid-template-columns: 1fr auto;
+  }
+
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    grid-template-columns: minmax(0, 1fr);
+  }
```

Comment:
Are these `@media` queries needed?

Looks like these styles are applied to a `div`, not the `<PageFilterBar>` itself, so that div is not going to be `display: grid` from what I can tell.

Looking around at other pages which use `<PageFilterBar>` it seems like we could drop the extra wrapped and just apply the width & max-width props to the PageFilterBar directly:

```
return (
  <StyledPageFilterBar>
    <ProjectPageFilter />
      ... snip ...

const StyledPageFilterBar = styled(PageFilterBar)`
  width: max-content;
  max-width: 100%;
  margin-bottom: ${space(1)};
`;
```

Reply:
Updated.